command/sub8_alarm/test_alarms/test_alarm_integration.py

```diff
@@ -46,15 +46,19 @@ def test_lose_thruster(self):
         fail_thruster_proxy(thruster_to_fail)
 
         time.sleep(0.1)
-        wrench_pub.publish(WrenchStamped(
-            # The actual force/torque is irrelevant, we still send a 0 command for every active thruster
-            wrench=Wrench(
-                force=Vector3(10.0, 10.0, 10.0),
-                torque=Vector3(10.0, 10.0, 10.0)
-            )
-        ))
+        give_up_time = time.time() + 1.5
```

Comment:
A comment here explaining your reasoning behind this change would be helpful. Why did you opt to use 1.5 as your timeout?